.travis.yml

```diff
@@ -1,36 +1,34 @@
----
 dist: xenial
-
 language: python
-
 python:
-  - "3.6"
-
+- '3.6'
 services:
-  - docker
-
+- docker
 env:
-  - mysql_root_username=root mysql_root_password='12345'
-
+  global:
+  - mysql_root_username=root
+  - mysql_root_password='12345'
+  - mysql_host=localhost
+  - mysql_db=devopsloft
 before_install:
-  - sudo apt-get install -y libmysqlclient-dev
-
+- sudo apt-get install -y libmysqlclient-dev
+- cp configuration.yml web_s2i/configuration.yml
 addons:
   apt:
     update: true
-
-cache:
-  apt: true`
-
 install:
-  - pip install -r requirements-dev.txt
-  - docker build -t devopsloft/devopsloft docker/.
-
+- pip install -r tests/requirements-test.txt
+- docker build -t devopsloft/mysql db_s2i/.
+- docker build -t devopsloft/devopsloft web_s2i/.
 before_script:
-  - docker run -d -p 80:80 -p 3306:3306 --name web devopsloft/devopsloft
-  - docker exec -it web bash -c 'while ! mysqladmin ping --silent; do sleep 3; done'
-
+- docker network create devopsloft_network
+- docker run -d --network devopsloft_network -p 3306:3306 --name db -e MYSQL_ROOT_PASSWORD=12345 -e MYSQL_DATABASE=devopsloft devopsloft/mysql
```

Comment:
You defined environment variables. The duplication is unnecessary.

Reply:
ooppss... fixing ...
BTW, how can I take advantage of configuration.yml in travis ?

Reply:
I suppose I can create an external script that read parameters from configuration.yml and run docker run. It can be useful also for manual start docker 
something like:
docker.sh start/stop web/db

Reply:
The values in the CI can be hard coded and can be different than the one used in the other environment. Don't invest in reading from a configuration file.

Reply:
Already invested, but there are inconsistent errors in Travis , can you look at my last build?

Reply:
done